WcaOnRails/lib/custom_i18n_scanner.rb

```diff
@@ -0,0 +1,67 @@
+require 'i18n/tasks/scanners/file_scanner'
+class WCAFileScanner < I18n::Tasks::Scanners::FileScanner
+  include I18n::Tasks::Scanners::RelativeKeys
+  include I18n::Tasks::Scanners::OccurrenceFromPosition
+
+  def active_record?(model)
+    # FIXME: Find a way to determine this dynamically ?
+    model != "contact"
+  end
+
+  def extract_model(key)
```

Comment:
I'd suggest `extract_model_name` as the current one could lead to assumption that the method returns the model class.